### DISCUSSION TITLE
Sync Calendar from outlook to Zimbra

### DISCUSSION THREAD

**Discussion by Mickalochard** [Z-Hub/Z-Push#115] [Q&A] [answered]
Hi everybody !

I use a Zimbra FOSS 9.0 with z-push 2.7.4 on the same machine and the ZImbra Backend 72. It works great for Emails sync but have a problem with Calendar. 

When I out an event in zimbra webmail, it sync on my mobile phone and also on outlook 2016, but when I try to put an event in outlook or in my mobile phone, it does not sync in my zimbra webmail. 

Question: does it work actually in this way ?

Thanks for your help

**Comment by matidau** [comment, marked as answer]
Version 73 of the Backend might solve some 9.0/10.0 issues. https://sourceforge.net/projects/zimbrabackend/files/Release73/

I believe that there are unreleased changes that may or may not help as well that you can get from https://sourceforge.net/p/zimbrabackend/code/HEAD/tree/zimbra-backend/branches/z-push-2/

You may get better responses on the Zimbra Backend Project https://sourceforge.net/p/zimbrabackend/discussion/

**Reply by Mickalochard** [reply]
Only for information, it works well with outlook on pc and outlook for mobile phones. When Use the Gmail app with exhange, the problem appears.
